templates/aws-dualstack/cluster_nodes/terraform/pools/corral.tf

```diff
@@ -1,10 +1,14 @@
-variable "corral_name" {} // name of the corral being created
-variable "corral_user_id" {} // how the user is identified (usually github username)
-variable "corral_public_key" {} // The corrals public key.  This should be installed on every node.
+variable "corral_name" {}        // name of the corral being created
+variable "corral_user_id" {}     // how the user is identified (usually github username)
+variable "corral_public_key" {}  // The corrals public key.  This should be installed on every node.
 variable "corral_private_key" {} // The corrals private key.  This should be installed on every node to be able to have root access, as aws does not allow this by default.
 
 variable "aws_access_key" {}
 variable "aws_secret_key" {}
+variable "aws_session_token" {
```

Comment:
Yash if it's optional I think we need to default it. The optional aspect should go here: https://github.com/rancherlabs/corral-packages/blob/main/templates/aws/nodes/manifest.yaml

Reply:
Also I don't see where it's used.

Reply:
@igomez06 for reference https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html
Since I was using a temporary token, I had to set this variable

Reply:
@l4zy0n3 But it's not really answering my question how are you setting this? And why isn't it in the manifest.yaml? Are you reading it as an env var? How would you set this in a Jenkins job?